### ISSUE TITLE
issue about the constant mask

### ISSUE THREAD

**Issue by ShileiCao** [zhaoshan2/pangu-pytorch#3]
what is the constants['const_h'] which associates with the operation “input = torch.cat((input, const_h), dim=1)” in PatchEmbedding layer

**Comment by zhaoshan2** [comment]
Hi, 
The constant_h is extracted from the original onnx file, and it contains some masks (.e.g., topography) of the input.
